btr-web/btr-main-app/.env.example

```diff
@@ -16,3 +16,6 @@ VUE_APP_LEGAL_API_VERSION_2="/api/v2"
 # pay API
 VUE_APP_PAY_API_URL="https://pay-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_PAY_API_VERSION="/api/v1"
+# BTR API
+VUE_APP_BTR_API_URL="https://btr-api-dev-nr4i2ahhhq-nn.a.run.app"
```

Comment:
can you add the env to the devops/vaults.env too? That way it'll be added in the deployment

`VUE_APP_BTR_API_URL="op://API/$APP_ENV/btr-api/BTR_API_URL"`